### DISCUSSION TITLE
1.10.0rc1

### DISCUSSION THREAD

**Discussion by sgillies** [Toblerity/Fiona#1433] [Announcements]
This is the first release candidate for 1.10.0.

### Changes

- Mutable item access to Feature, Geometry, and Properties instances has been restored (reported in #1430). This usage should be avoided as instances of these classes will be immutable in a future version.
- The setup.cfg duplicates project configuration in pyproject.toml and has been removed.

### Packaging

- PyPI wheels include GDAL 3.9.2.

Please see the CHANGES file for a more detailed enumeration of changes since version 1.10b1. Unless showstopper bugs are found in this release, expect a final 1.10.0 release in two weeks.

<hr /><em>This discussion was created from the release <a href='https://github.com/Toblerity/Fiona/releases/tag/1.10.0rc1'>1.10.0rc1</a>.</em>

**Comment by EwoutH** [comment]
Thanks a lot for the release candidate, always much appreciated!

Could we get Python 3.13 wheels for 1.10?

**Reply by sgillies** [reply]
That's the plan.

**Reply by sgillies** [reply]
`fiona-1.10.0rc1-cp313-cp313-macosx_10_15_x86_64.whl`, `fiona-1.10.0rc1-cp313-cp313-macosx_11_0_arm64.whl`, and `fiona-1.10.0rc1-cp313-cp313-manylinux_2_17_x86_64.manylinux2014_x86_64.whl` have been uploaded to PyPI. One for Windows will be ready in an hour or so.

Thanks for the assist @EwoutH !

**Reply by EwoutH** [reply]
Thanks!
